### PR TITLE
Feature/topic area

### PR DESCRIPTION
**Pull request by GeorgDangl**
As discussed in the telecon on 10.10.2016

### CODE REVIEW

**Comment by GeorgDangl** [comment]
See https://github.com/BuildingSMART/BCF-XML/pull/115 in the BCF XML repository

File: README.md [hunk 1]
```diff
@@ -619,6 +619,12 @@ Project extensions are used to define possible values that can be used in topics
             "BIM-Manager@example.com",
             "bob_heater@example.com"
         ],
+        "topic_area"_ [
```
Comment (GeorgDangl) [inline README.md]: `_` should be `:`

**Comment by jasollien** [comment]
If merging, also remember to update the Events

**Comment by GeorgDangl** [comment]
This can now be handled with the custom properties, #219 (and the PR #222)

**Pull request closed without merging**